### PR TITLE
fix irregular days alignment issues

### PR DESCRIPTION
**Pull request by Joosakur**

### CODE REVIEW

File: frontend/src/citizen-frontend/calendar/reservation-modal/TimeInputs.tsx [hunk 1]
```diff
@@ -177,10 +175,11 @@ const ReadOnlyDay = React.memo(function ReadOnlyDay({
         </>
       )
     case 'holiday':
+      // doesn't go here, bug?
       return (
-        <FixedSpaceRow fullWidth>
+        <FixedSpaceRow fullWidth alignItems="center">
           <LeftCell>{label}</LeftCell>
-          <MiddleCell textOnly>{i18n.calendar.holiday}</MiddleCell>
+          <MiddleCell>{i18n.calendar.holiday}</MiddleCell>
```
Comment (Joosakur) [inline frontend/src/citizen-frontend/calendar/reservation-modal/TimeInputs.tsx:182]: I added a holiday to db but still it didn't go to this branch (was case undefined) and didn't say pyhäpäivä on the UI

Comment (akheron) [reply frontend/src/citizen-frontend/calendar/reservation-modal/TimeInputs.tsx:182]: I'll fix in the next PR

**Review by akheron — APPROVED** [review]
🏅

File: frontend/src/citizen-frontend/calendar/reservation-modal/TimeInputs.tsx [hunk 2]
```diff
@@ -177,10 +175,11 @@ const ReadOnlyDay = React.memo(function ReadOnlyDay({
         </>
       )
     case 'holiday':
+      // doesn't go here, bug?
```
Comment (akheron) [inline frontend/src/citizen-frontend/calendar/reservation-modal/TimeInputs.tsx:178]: Remove this comment before merging 😇

Comment (Joosakur) [reply frontend/src/citizen-frontend/calendar/reservation-modal/TimeInputs.tsx:178]: ooh yeah thanks xD

**Pull request merged**